tests/datadriven/test_variants.py

```diff
@@ -192,6 +193,14 @@ def _verifyVariantCalls():
                     self.assertEqual(str(alt1), str(alt2))
             else:
                 self.assertEqual(gaVariant.alternate_bases, alt)
+            if not gaVariant.filters_applied:
```

Comment:
Thanks, I made this issue so that we can circle back to it. https://github.com/ga4gh/server/issues/1589